pkg/volume/gcepd/attacher.go

```diff
@@ -150,6 +199,17 @@ func (attacher *gcePersistentDiskAttacher) WaitForAttach(spec *volume.Spec, devi
 	}
 
 	pdName := volumeSource.PDName
+
+	if runtime.GOOS == "windows" {
```

Comment:
Should this also retry and honor the timeout?

Reply:
talked to Saad, and we think this loop should be removed. Open an issue to track this #74765